## 📋 Quality Gate Results

| Check | Status |
|-------|--------|
| Knip | ✅ success |
| Lint | ❌ failure |
| Build | ✅ success |
| Infra Tests | ❌ skipped |
| Unit Tests | ❌ skipped |
| Component Tests | ❌ skipped |
| Perf Tests | ❌ skipped |
| Visual Tests | ❌ skipped |


### ❌ Lint Failure Details
```text

> hrm@0.23.0 lint /home/runner/work/hrm/hrm
> eslint app/ components/ constants/ context/ hooks/ lib/ services/ tests/ types/ utils/ server.ts proxy.ts --cache


/home/runner/work/hrm/hrm/tests/unit/scripts/create-review-issues.test.ts
  157:22  error  Delete `;`  prettier/prettier
  161:57  error  Delete `;`  prettier/prettier
  164:18  error  Delete `;`  prettier/prettier
  165:9   error  Delete `;`  prettier/prettier

✖ 4 problems (4 errors, 0 warnings)
  4 errors and 0 warnings potentially fixable with the `--fix` option.

 ELIFECYCLE  Command failed with exit code 1.
```
### ❌ Infrastructure Test Failure Details
```text
Log file not found.
```
### ❌ Unit Test Failure Details
```text
Log file not found.
```
### ❌ Component Test Failure Details
```text
Log file not found.
```
### ❌ Visual Test Failure Details
```text
Log file not found.
```
### ❌ Performance Test Failure Details
```text
Log file not found.
```
⚠️ **Some checks failed.** Full logs available in [workflow artifacts](https://github.com/arii/hrm/actions/runs/20767268963).

---
> Report generated for commit: `2f96e48a64d10a7b992aae4038e7cc76b504df35`

--- a/.github/workflows/create-issues-from-command.yml
+++ b/.github/workflows/create-issues-from-command.yml
@@ -24,9 +24,12 @@ jobs:
     secrets:
       gh_token: ${{ secrets.ARI_PAT }}
 
-  create-issues-from-comment:
+  prepare-comment-run:
     if: github.event_name == 'issue_comment' && github.event.issue.pull_request && contains(github.event.comment.body, '/create-issues')
     runs-on: ubuntu-latest
+    outputs:
+      pr_number: ${{ github.event.issue.number }}
+      run_id: ${{ steps.find_run.outputs.run_id }}
     steps:
       - name: 'Get PR branch name'
         id: get_pr
@@ -48,10 +51,12 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.ARI_PAT }}
 
-      - name: 'Create issues from review'
-        uses: ./.github/workflows/reusable-create-review-issues.yml
-        with:
-          pr_number: ${{ github.event.issue.number }}
-          run_id: ${{ steps.find_run.outputs.run_id }}
-        secrets:
-          gh_token: ${{ secrets.ARI_PAT }}
+  create-issues-from-comment:
+    if: github.event_name == 'issue_comment' && github.event.issue.pull_request && contains(github.event.comment.body, '/create-issues')
+    needs: prepare-comment-run
+    uses: ./.github/workflows/reusable-create-review-issues.yml
+    with:
+      pr_number: ${{ needs.prepare-comment-run.outputs.pr_number }}
+      run_id: ${{ needs.prepare-comment-run.outputs.run_id }}
+    secrets:
+      gh_token: ${{ secrets.ARI_PAT }}

--- a/tests/unit/scripts/create-review-issues.test.ts
+++ b/tests/unit/scripts/create-review-issues.test.ts
@@ -154,15 +154,15 @@ describe('create-review-issues script', () => {
       ;(execSync as jest.Mock).mockImplementation((command: string) => {
         if (command.startsWith('gh label list')) {
           // Return an empty array for the label list to simulate no existing labels
-          return '[]';
+          return '[]'
         }
         if (command.startsWith('gh issue create')) {
           // Return a URL for the issue creation
-          return 'https://github.com/test/repo/issues/1';
+          return 'https://github.com/test/repo/issues/1'
         }
         // Return an empty string for any other command to avoid unexpected behavior
-        return '';
-      });
+        return ''
+      })
 
       ghClient.createIssue(issue, context)
 

